src/openrct2/ride/TrackPaint.h

```diff
@@ -546,10 +546,13 @@ void DrawSBendRightSupports(
     PaintSession& session, MetalSupportType supportType, uint8_t sequence, Direction direction, int32_t height,
     int32_t specialA, int32_t specialB);
 
-using TRACK_PAINT_FUNCTION = void (*)(
+using TRACK_PAINT_FUNCTION = void (&)(
```

Comment:
```suggestion
using TrackPaintFunction = void (&)(
```
I think its time to make it match our code style.